trnx.go

```diff
@@ -190,14 +190,18 @@ func (trx *optimisticTransactionImp) Commit(actionsIter iter.Iter[action.Action]
 	}
 	isBlindAppend := !dependsOnFiles && onlyAddFiles
 
+	opAny := make(map[string]any, len(op.Parameters))
+	for k, v := range op.Parameters {
+		opAny[k] = v
+	}
```

Comment:
please check why we need to convert from map[string]string as operation should hold map[string]any